src/main/kotlin/verticles/RESTVerticle.kt

```diff
@@ -166,18 +168,24 @@ class RESTVerticle(
                         }
 
                     } else {
-                        println("getElements: ${ar.result().statusCode()} : ${ar.result().statusMessage()}")
+                        println("Error on requesting elements: ${ar.result().statusCode()} : ${ar.result().statusMessage()}")
+                        printElementIds(elementIds)
                         myError()
                     }
                 } else {
                     println("Query Root Element failed: ${ar.cause().message}")
+                    printElementIds(elementIds)
                     myError()
                 }
 
             }
 
     }
 
+    private fun printElementIds(elementIds: List<String>) {
+        println("  Element IDs: ${elementIds.joinToString(",")}")
```

Comment:
Does this print it out in a form that can be simply copy-pasted into Swagger/Postman?

Reply:
Yes, exactly. You can copy the list after 'element IDs' and use it in the body of the corresponding POST request.